### ISSUE TITLE
Shows with similiar name get mixed up

### ISSUE THREAD

**Issue by psonnosp** [razzeee/script.trakt#499]
So I have the show [Backstrom](https://trakt.tv/shows/backstrom) unwatched in my Kodi library with the correct TheTVDB and IMDb ID in the NFO.

I then started to watch the Swedish show [Bäckström](https://trakt.tv/shows/backstrom-2020), outside of Kodi, and marked them as watched on Trakt and now the episodes of Backstrom get marked as watched.

Shouldn't the Trakt script use the ID if there's one available instead for going for part of the show name?

**Comment by razzeee** [comment]
Yes, can you please post the NFO file? There were changes in the format some months ago.

**Comment by psonnosp** [comment]
It should be up to date using the uniqueid-tag as far as I know.
https://paste.kodi.tv/guxujedoqo

**Comment by razzeee** [comment]
Interesting, that looks correct, do you remember if your settings changed from the defaults?

**Comment by psonnosp** [comment]
I've turned on 'Sync collection on library update or cleaning' and 'Remove deleted movies/TV shows from Trakt collection' but that's it.

**Comment by razzeee** [comment]
I think there is a setting, that allows trakt to fallback to title matching, can you check if the issue happens, if you deactivate that?

**Comment by psonnosp** [comment]
That's off by default, I think, but it's not enabled now so that's not the issue.

**Comment by razzeee** [comment]
The best shot at figuring this out might be a comlpete log https://github.com/trakt/script.trakt#creating-logfiles
